manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml

```diff
@@ -538,4 +538,3 @@ spec:
       kind: HiveTable
       name: hivetables.metering.openshift.io
       version: v1
-
```

Comment:
Add this line back and verify should pass

Reply:
Remove the indentation from this line - you can configure your editor to do this by default (removing spaces from an empty line)